modules/snooker/src/test/ts/module/ephox/snooker/test/Assertions.ts

```diff
@@ -192,13 +241,14 @@ const checkDelete = (
     // the result of a delete operation can be by definition the deletion of the table itself.
     // If that is the case our table should not have any parent element because has been removed
     // from the DOM
-    assert.eq(false, Traverse.parent(table).isSome(), 'The table was expected to be removed from the DOM');
+    Assertions.assertEq(label + ': The table was expected to be removed from the DOM', false, Traverse.parent(table).isSome());
+    // assert.eq(false, Traverse.parent(table).isSome(), 'The table was expected to be removed from the DOM');
```

Comment:
Should this be commented out or removed?

Reply:
Fixed